src/common/common.js

```diff
@@ -516,14 +516,14 @@ const MessageHandler = (function () {// eslint-disable-line no-unused-vars
             elMessage.textContent = localizedString;
         }
 
-        if (elMessage.classList.contains("dismissible")) {
+        if (typeof args[1] === "boolean" && args[1]) {
             // add an icon which dismisses the message if clicked
             const iconDismiss = document.createElement("span");
```

Comment:
Okay, it creates this button – but it does not remove it (or at least make it invisible) when another message without "dismissable" set to true is shown.

As such maybe generally do it in another way: Add the HTML to the html file and only hide/unhide it when needed. This has two other advantages:
* if a message may never get a dismissable button, because of the way it is shown or so, it could just not be added to the HTML (such a case would be the Popup QR code error, IMHO). So in the JS you need to check for the existence of that element!
* That would also make customization easier.

Reply:
But all contents of the message box are replaced by setting `elMessage.textContent = localizedString;` [here](https://github.com/rugk/offline-qr-code/pull/64/files#diff-ccdc4da75e7ac1061a8cc0851e3c7122L516)...